example/lib/peripheral_list/components/peripheral_category_icon.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:blemulator_example/common/components/title_icon.dart';
+import 'package:blemulator_example/common/components/title_image_icon.dart';
+import 'package:blemulator_example/model/ble_peripheral.dart';
+import 'package:blemulator_example/styles/custom_colors.dart';
+import 'package:flutter/material.dart';
+
+class PeripheralCategoryIcon extends StatelessWidget {
+  final BlePeripheralCategory _peripheralCategory;
+
+  PeripheralCategoryIcon(this._peripheralCategory);
+
+  @override
+  Widget build(BuildContext context) {
+    if (_peripheralCategory == BlePeripheralCategory.sensorTag) {
+      return TitleImageIcon(
```

Comment:
What's the benefit of having TitleIcon and TitleImageIcon if both are used only here and only call the constructor of super type?

Reply:
I can see them being used as part of more widgets in future features, and that's why I've extracted them to separate widgets.

They call the constructor of super type with limited set of properties (ie: color and IconData or asset), with fixed size (defined in `custom_sizes.dart`) to help maintain consistent design throughout the app.